resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -733,6 +757,14 @@ def start(self, data: Data, listener_connection: Tuple[str, str, str]):
             processing_container_image, mapper
         )
 
+        # Create secrets.
+        secrets_name = f"secrets-{data.pk}-{random_postfix}"
```

Comment:
I would prefer `resolwe` prefix instead of `secrets`, as it better fits our existing cluster resource naming conventions. Our pods and jobs are not prefixed with `pod-`, or `job-`, but with `resolwe-`.

Reply:
This was actually already accepted as a part of a different PR.